manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
@saliceti yes, that's correct. I forgot about it..but anyway...I've got commit with new CA for ipsec so I'm gonna test it now.